tools/castle-editor/code/framedesign.pas

```diff
@@ -2157,6 +2268,7 @@ procedure TDesignFrame.MenuItemViewportCameraCurrentFromInitialClick(
     V.Camera.InitialDirection,
     V.Camera.InitialUp);
   ModifiedOutsideObjectInspector;
+  RecordUndo('MenuItemViewportCameraCurrentFromInitialClick'); //whatever that means?
```

Comment:
Should be `RecordUndo('Camera Current := Initial');`

( Actually this doesn't change the persistent state now; only "just in case this will change in future" we call here `ModifiedOutsideObjectInspector` and so also `RecordUndo`. I see that your undo mechanism will already smartly ignore such no-change, and that's cool. )

Reply:
Thanks, fixed!